src/components/Kv/KvCurrencyInput.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+	<input
+		type="text"
+		v-model="displayValue"
+		@blur="isInputActive = false"
+		@focus="isInputActive = true"
+		placeholder="$"
+	>
+</template>
+
+<script>
+export default {
+	props: {
+		value: {
+			type: Number,
+			default: 0,
+		}
+	},
+	data() {
+		return {
+			isInputActive: false
+		};
+	},
+	computed: {
+		displayValue: {
+			get() {
+				if (this.isInputActive) {
+					// Cursor is inside the input field. unformat display value for user
+					return this.value.toString();
+				}
+				// User is not modifying now. Format display value for user interface
+				return `$ ${this.value.toFixed(2).replace(/(\d)(?=(\d{3})+(?:\.\d+)?$)/g, '$1,')}`;
```

Comment:
I think you could use numeral in all of these cases to flip back and forth between the various formats + values...